crawler/src/main/java/de/unidisk/crawler/rest/dto/keyword/CreateKeywordDto.java

```diff
@@ -0,0 +1,30 @@
+package de.unidisk.crawler.rest.dto.keyword;
+
+public class CreateKeywordDto {
```

Comment:
Die Dto's haben doch alle ein ähnliches Schema, wäre da nicht eine Vererbungsstruktur sinnvoll?

Reply:
Denke, dass Vererbung hier erstmal keinen großen Vorteil bringt. Bei Projekt könnten wir statt `name` auch `titel` als Namen benutzen, wodurch die Verberung dann schon wieder hinfällig wäre.

Reply:
Aber es sind doch ungefähr 3/4 Dto's gewesen, die alle ähnliche Methoden haben. Das ist doch quasi ein Interface, was die alle bedient.

Reply:
In wie fern Methoden? Die Dtos selbst dienen ja nur zur Repräsentation der Inputdaten von der API.